app/resources/add_note_base.html.template

```diff
@@ -129,37 +134,13 @@
           <div class="end-multi-columns"></div>
         {% endif %}
       </div>
-    </div>
   {% endif %}
 
-
-
-  <div class="section">
-    <h2>Message</h2>
-    <h3 class="mandatory text">
-      {% trans "Message (required)" %}
-    </h3>
-    <div class="hint">
-      {% if not markdup %}
-        {% trans "A message for this person or others seeking this person" %}
-      {% else %}
-        {% trans "Please explain why you think these are the same person" %}
-      {% endif %}
-    </div>
-    <div class="field">
-        <textarea name="text" id="text"
-          class="long-text-input"
-          rows=5>{{params.text}}</textarea>
-    </div>
-    <div class="end-multi-columns"></div>
-  </div>
   
-  {% if not markdup %}
-
-    
+  {% if not markdup %}    
     {% if env.enable_photo_upload %}
       <div class="photo section">
-        <h2>Photo</h2>
+        <h3>Photo</h3>
```

Comment:
The <h3> part is actually hiding inside the expanding part as a subtitle(like the following photo), and I am using the <h2> part for the center title like Identify information/Source of record etc...

![note with expand2](https://cloud.githubusercontent.com/assets/15961861/17657633/e96d29da-62ff-11e6-9342-bafcd81aba40.png)